conf/modules.config

```diff
@@ -979,56 +987,36 @@ process {
         ]
     }
 
-    withName: 'ANNOTATION_BGZIPTABIX' {
+    withName: "NFCORE_SAREK:SAREK:ANNOTATE:.*:TABIX_BGZIPTABIX" {
         publishDir       = [
                 mode: params.publish_dir_mode,
                 path: { "${params.outdir}/annotation/${meta.id}/${meta.variantcaller}" },
                 pattern: "*{gz,gz.tbi}"
         ]
     }
-}
 
-if (params.tools && (params.tools.contains('snpeff') || params.tools.contains('merge'))) {
```

Comment:
I guess we still need to find a way for most processes anyways to avoid printing.

Reply:
This will be solve by Nextflow at some point

Reply:
Or should anyway